### PR TITLE
Update README.md to add Install verification instructions

### PR DESCRIPTION
**Pull request by ktangsali**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,13 +32,33 @@ The recommended method for installing the latest version of Modulus is using PyP
 pip install nvidia-modulus
 ```
 
+The installation can be verified by running a simple python code snippet as shown below:
+```
+>>> import torch
```
[the remark this answers is not in the document]
Comment (ktangsali) [reply README.md:38]: Good point. Currently we don't package pytests with the pip wheel so something like that is not possible, but I added `python` command.

**Review by NickGeneva — APPROVED** [review]

**Pull request merged**